airflow/settings.py

```diff
@@ -140,10 +155,11 @@ def configure_orm(disable_connection_pool=False):
         except conf.AirflowConfigException:
             pool_recycle = 1800
 
-        log.info("settings.configure_orm(): Using pool settings. pool_size={}, "
-                 "pool_recycle={}, pid={}".format(pool_size, pool_recycle, os.getpid()))
+        log.info("settings.configure_orm(): Using pool settings. pool_size={}, max_overflow={}"
```

Comment:
Missing , between max_overflow and pool_recycle

Reply:
Ohh, thanks for your comment :)